test/test_commander.py

```diff
@@ -0,0 +1,99 @@
+import os, shutil
+import unittest
+from unittest.mock import patch
+
+from sss import commander
+from sss.dataclasses import *
+
+
+# mocked
+import soundfile
+
+class TestCommander(unittest.TestCase):
+        
+    def test_extract(self):
+        # given
+        method = "nmf"
+        extract_parameters = ExtractParams(
+            input_path="maanam-test.wav",
+            instruments=[Instrument("vocals")],
+            reverse=False,
+            quality="fast",
+            max_iter=1)
+        
+        # when
+        actual_results = commander.extract(method, extract_parameters)
```

Comment:
Na tym mi się niestety testy wywalają, nie mam pojęcia dlaczego, ale przy odpalaniu testów nie potrafi poprawnie zinterpretować ścieżki (chyba wymaga ścieżek bezwzględnych). Jednocześnie wersja konsolowa i gui mi działają.
Piszę to bardziej jako ciekawostkę, nie wiem, czy chcemy coś z tym robić. Jeśli u Ciebie działa to dobrze :)

Reply:
Więc przez to nie wszystko przechodzi u mnie

Reply:
Sprawdzę to w najbliższym czasie